docs/src/ensemble_kalman_inversion.md

```diff
@@ -115,7 +115,11 @@ The EKI algorithm drives the initial ensemble, sampled from the prior, towards t
 ```julia
 using Statistics
 
-θ_optim = mean(get_u_final(ekiobj), dims=2)
+θ_optim = mean(get_u_final(ekiobj), dims=2) # the optimal parameter
```

Comment:
We can change this to the new getter `get_u_mean_final(ekiobj)` if we merge #208 before.